enforcer/enforcer.go

```diff
@@ -160,7 +160,8 @@ func (e *Enforcer) fetchImagePullSecrets(pod *corev1.Pod) ([]*imagePullSecret, e
 }
 
 func (e *Enforcer) evaluatePolicy(log *zap.Logger, response *v1.AdmissionResponse, imageName string, pullSecrets []*imagePullSecret) bool {
-	log = log.With(zap.String("image", imageName))
+	ctx := context.Background()
+	log = log.With(zap.String("image", imageName)).With(zap.String("policy group", e.config.PolicyGroup))
```

Comment:
```suggestion
	log = log.With(zap.String("image", imageName)).With(zap.String("policyGroup", e.config.PolicyGroup))
```

Nitpick, but with the production config the log format will be json and it's a little awkward to have the space there, for instance when writing log queries.